packages/react-ui/src/widgets/PieWidgetUI/PieWidgetUI.js

```diff
@@ -215,15 +228,31 @@ function PieWidgetUI({
     }
   };
 
+  const handleClearSelectedCategories = () => {
+    onSelectedCategoriesChange([]);
+  };
+
   if (isLoading) return <PieSkeleton height={height} />;
 
   return (
-    <ReactEcharts
-      option={options}
-      onEvents={onEvents}
-      lazyUpdate={true}
-      style={{ maxHeight: height }}
-    />
+    <>
+      <OptionsBar container>
+        <Typography variant='caption' color='textSecondary'>
+          {selectedCategories.length ? selectedCategories.length : 'All'} selected
```

Comment:
no translations in c4r?

Reply:
There is a draft PR to add them, but actually we don't have :( 

https://github.com/CartoDB/carto-react/pull/727